dll/win32/shdocvw/winlist.cpp

```diff
@@ -91,7 +481,16 @@ EXTERN_C
 VOID WINAPI
 WinList_Terminate(VOID)
 {
-    FIXME("()\n");
+    TRACE("()\n");
+
+    // Probably, we have nothing to do with "Internet Explorer" here. We are not Windows.
```

Comment:
What's the relation with Internet Explorer here, actually?

Reply:
`WinList` might contains IE info.
IE info has many shared info.
This function on Windows has some IE-related function calls.
ReactOS doesn't have IE.

Reply:
Windows does like this:
```txt
BOOL WinList_Init()
{
  if ( GetSharedServiceMode(2) == 3 )
    IEDDE_Initialize(); // IE-related
 ...
}
void WinList_Terminate()
{
  IEDDE_Uninitialize(); // IE-related
  if ( g_pcfWinList )
  {
    SHSharedServicesRevoke(0);
    g_pcfWinList->Release();
    g_pcfWinList = NULL;
  }
  IERevokeClassFactoryObject(); // IE-related
  CUrlHistory_CleanUp(); // IE-related
}
```